src/SharpGLTF.Cesium/SharpGLTF.Cesium.csproj

```diff
@@ -20,4 +20,8 @@
     <Folder Include="Schema2\Generated\" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="OneOf" Version="3.0.263" />
```

Comment:
That's an interesting one....  my concern is dragging a reference that just adds a few classes that can be easily added to the basecode.

I'll think about it later... for now, no action

Reply:
yeah it's a little library that reduced a lot of my code, hope it doesn't give a conflict somewhere. Maybe we can also use it in the code generator project?

Reply:
The code generation library can have whatever dependencies we might require to simplify and ease the code generation because it is not being distributed.

For distributable packages the less dependencies, the better, but for now it is fine.